lang/en.js

```diff
@@ -105,6 +105,9 @@ export default {
 	"components.switch.visibleWithPeriod": "Visible.",
 	"components.switch.hidden": "Hidden",
 	"components.switch.conditions": "Conditions must be met",
+	"components.table-col-sort-button.addSortOrder": "Click to add sort order",
+	"components.table-col-sort-button.changeSortOrder": "Click to change sort order",
```

Comment:
The spec has "Click" but I'm wondering if it should be something like "Select" instead